app/controllers/lcms/engine/api/resources_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Lcms
+  module Engine
+    module Api
+      class ResourcesController < BaseController
+        def index
+          resources = Resource
+
+          # format for the query string is: ?link_updated_after={link_path}:{timestamp}
+          if params[:link_updated_after]
+            link_path, timestamp = params[:link_updated_after].split(':')
+
+            resources = resources.where_link_updated_after(link_path, Time.at(timestamp.to_i))
+          end
+
+          render json: resources.all.as_json
```

Comment:
Do we need special serializer for that?

Reply:
I don't think so

Reply:
Ok, just thought maybe we don't need to fetch all fields we have now at `ResourceSerializer` for this request but have `links` field.